test_conformance/printf/test_printf.cpp

```diff
@@ -144,7 +144,10 @@ static int getTempFileName()
 static int acquireOutputStream()
 {
     int fd = streamDup(fileno(stdout));
-    freopen(gFileName,"w",stdout);
+    if (!freopen(gFileName, "w", stdout))
+    {
+        log_error("Error while redirection stdout to file");
```

Comment:
Probably worth throwing an exception or returning a negative value and updating all using code to check the error.